### PR TITLE
Bump to version 0.10.2

### PR DESCRIPTION
**Pull request by paulosman**

### CODE REVIEW

**Review by vreynolds — APPROVED** [review]
🎖️

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Release v0.10.2 (2020-12-23)
+
+- Bump otel dependencies from 0.13.0 to 0.15.1
```
Comment (MikeGoldsmith) [inline CHANGELOG.md:3]: This should include otel updates too

**Review by MikeGoldsmith — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (paulosman) [reply CHANGELOG.md:3]: @MikeGoldsmith the only dependabot update was otel-contrib from 0.15.0 to 0.15.1.

Comment (MikeGoldsmith) [reply CHANGELOG.md:3]: Ah, okay - no problem 👍

**Review by MikeGoldsmith — APPROVED** [review]

**Pull request merged**